PixaPicker/PixaDataCoordinator.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  PixaDataCoordinator.swift
+//  PixaPicker
+//
+//  Created by Thomas Caraway on 11/22/19.
+//  Copyright © 2019 Thomas Caraway. All rights reserved.
+//
+
+import Foundation
+
+class PixaDataCoordinator {
+    
+    var delegate: PixaDataCoordinatorDelegate?
+    var cellImageURLs = [URL]()
+    var currentPageNumber = 1
+    let maxImagesPerPage = 20
+    
+    func getNextPage(withCurrentSearchText: String){
+        currentPageNumber += 1
```

Comment:
one thing to think about: how do we stop `getNextPage` from accidentally getting double called?  
How we we make sure if, when `getNextPage` is called twice really fast, they're both returned in order?